tests/test_adapters/test_finish_reason_interceptor.py

```
@@ -0,0 +1,134 @@
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for the finish_reason reconciliation interceptor."""
+
+from __future__ import annotations
+
+import logging
+
+from nemo_evaluator.adapters.interceptors.finish_reason import Interceptor
+from nemo_evaluator.adapters.types import AdapterRequest, AdapterResponse, InterceptorContext
+
+
+def _make_pair(request_body, response_body):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_make_pair`

(ANN202)